defaults/main.yml

```diff
@@ -36,8 +36,8 @@ openio_replicas: 3
 # this `.{}` will be filled by `format_location()`
 openio_location: "{{ ansible_hostname }}.{}"
 
-openio_minimal_score_for_volume_admin_bootstrap: 50
-openio_minimal_score_for_directory_bootstrap: 50
+openio_minimal_score_for_volume_admin_bootstrap: 10
```

Comment:
Why ?

Reply:
Not really related, but in some cases scores take too long to reach 50, so lowering them is a good idea.

Reply:
@racciari @vincent-legoll can you validate this change ?